kpi/utils/iterators.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+from typing import Union
+
+
+def compare(iter_a: Union[list, tuple], iter_b: Union[list, tuple]):
+
+    if not isinstance(iter_a, type(iter_b)):
+        return False
+
+    if len(iter_a) != len(iter_b):
+        return False
+
+    return iter_a == iter_b
+
+
+def to_int(iterator: list, unique: bool = True):
+
+    coerced = map(int, iterator)
+    if unique:
+        return list(set(coerced))
+    return list(coerced)
```

Comment:
is it worthwhile to have a function for this? :smiling_imp: i'll admit that your way does save a bit of typing :smile: 
```python
to_int(iterable)
[int(x) for x in iterable]

to_int(iterable, unique=True)
[int(x) for x in set(iterable)]
```

Reply:
:sunglasses: https://chat.kobotoolbox.org/#narrow/stream/4-KoBo-Dev/topic/Python.20coding.20style/near/45031
[but @noliveleger raised the more-important point that sometimes generators are passed to `to_int()` and wrongly coerced to lists—they should remain generators]

Reply:
Done in https://github.com/kobotoolbox/kpi/pull/2900/commits/3385ea9379a30bc0a139ded3220d1a09c13e93b6